tensorflow_encrypted/tensor/int32.py

```diff
@@ -111,6 +115,19 @@ def concat(x: List['Int32Tensor'], axis: int) -> 'Int32Tensor':
 
         return Int32Tensor.from_native(tf.concat(backing, axis=axis))
 
+    def binarize(self) -> NativeTensor:
+        """Computes bit decomposition of tensor
+         tensor: ndarray of shape (x0, ..., xn)
+        returns: a binary tensor of shape (x0, ..., xn, bits) equivalent to tensor
+        """
+        bitwidths = tf.range(bits, dtype=tf.int32)
+        for i in range(len(self.shape)):
```

Comment:
think this could be replaced by `tf.reshape(bitwidths, [1]*len(..))`, don't know if it's faster though